src/util/generatePreview.js

```diff
@@ -39,6 +39,63 @@ const errorHandlerScript = `(${function() {
   };
 }.toString()}());`;
 
+const elementHighlighterScript = `(${function() {
```

Comment:
The amount of code in here is getting pretty intense and it’s making me want to find a way to organize it properly. I think my ideal situation would be to have something like `src/previewFrame/index.js` which then can load submodules like `src/previewFrame/elementHighlighter.js` all with nice `imports` and everything.

I *think* the cleanest(?) way to do this would be a Webpack loader that evaluates the source file as an entry point and does a full-on Webpack compilation of it. As far as I can tell such a thing does not currently exist, probably because the concept is pretty diabolical.

It’s not clear to me that the proposed reorganization should block this PR, though.